docs/user-tutorial/monitor.md

```diff
@@ -0,0 +1,54 @@
+---
+id: monitor
+---
+
+# Monitor
+
+SuperBench provides a `Monitor` module to collect the system metrics and detect the failure during the benchmarking. Currently this monitor supports CUDA platform only. Users can enable it in the config file.
+
+## Configuration
```

Comment:
move it to superbench-config.mdx?

Reply:
superbench-config.mdx has already contained the Configuration part.